amivapi/media.py

```diff
@@ -10,6 +10,7 @@
 
 """
 
+from bson import tz_util
```

Comment:
Also could you make add a test for this problem? Or I'll try to add it later if I have time.